sbom_generation.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+
+# This OCI DevOps build specification file [1] generates a Software Bill of Materials (SBOM) of the repository.
+# The file is needed to run checks for third-party vulnerabilities and business approval according to Oracle’s GitHub policies.
+# [1] https://docs.oracle.com/en-us/iaas/Content/devops/using/build_specs.htm
+
+version: 0.1
+component: build
+timeoutInSeconds: 1000
+shell: bash
+
+steps:
+  - type: Command
+    name: "Install npm"
+    command: |
+      # Install npm version 8.19.4
+      npm i -g npm@v8.19.4
+  - type: Command
+    name: "Install node-cli & unit-testing packages"
+    command: |
+      npm install --ignore-scripts
+  - type: Command
+    name: "Install cyclonedx-node-npm package"
+    command: |
+      npm install --ignore-scripts -g --save-dev @cyclonedx/cyclonedx-npm
+  - type: Command
+    name: "Run cyclonedx-node-npm package"
+    command: |
+      # For more details, visit https://github.com/CycloneDX/cyclonedx-node-npm/blob/main/README.md
+      npx @cyclonedx/cyclonedx-npm --omit dev --output-format JSON --output-file artifactSBOM1.json --spec-version 1.4
+  - type: Command
+    name: "Install vscode extension package"
+    command: |
+      cd packages/vscode-extension && npm install --ignore-scripts
+  - type: Command
+    name: "Install cyclonedx-node-npm package"
+    command: |
+      npm install --ignore-scripts -g --save-dev @cyclonedx/cyclonedx-npm
```

Comment:
I doubt that you need to  install `@cyclonedx/cyclonedx-npm` as it was already globally installed before.